src/test/resources/manual-test/readme.md

```diff
@@ -0,0 +1,14 @@
+JSON files for manual tests
+---------------------------
+
+* Launch a virtual machine with `vagrant up` in the root of the project
+* Create a deposit, for details see `docs/api.html`
+* Use the returned UUID to replace XXX in the following command.
+
+    curl -i  -H 'Content-Type: text/plain'  --data-binary "@XXX.json" -X POST -u user001:user001 'http://test.dans.knaw.nl:20190/deposit/XXX/datasetmetadata'
```

Comment:
Maybe you can add some more things here about:

1. what should the outcome of this call be?
2. what are we testing here/what is the purpose of this test?
3. you would do future you or other developers a nice service if you would spell out the commands for creating a deposit instead of pointing them to the documentation.
4. make a link to the documentation instead.